ijPlugin/src/main/kotlin/org/jetbrains/research/tasktracker/config/DefaultConfigsFactory.kt

```diff
@@ -24,6 +26,8 @@ object DefaultConfigsFactory {
         ScenarioConfig.CONFIG_FILE_PREFIX,
         WebCamTrackingConfig.CONFIG_FILE_PREFIX,
         MainPageContentConfig.CONFIG_FILE_PREFIX,
+        FinalPageContentConfig.CONFIG_FILE_PREFIX,
```

Comment:
I'm not sure, but maybe we can somehow connect this list and the configs from the MainTaskTrackerConfig? Since it's not obvious that we need to remember about several places if we decide to add a new config

Reply:
Unfortunately, we cannot do this for several reasons. Firstly, because at the time DefaultConfigFactory is being executed, the other configurations in MainTaskTrackerConfig are not yet defined. Secondly, we cannot inherit static fields, and therefore, given this architecture, it seems impossible to me.

I suggest discussing options to redesign the architecture and eliminate this issue in our upcoming call.